lib/matplotlib/backends/backend_webagg_core.py

```diff
@@ -153,6 +153,10 @@ def draw(self):
         finally:
             self.manager.refresh_all()  # Swap the frames.
 
+    def blit(self, bbox=None):
+        self._png_is_old = True
+        self.manager.refresh_all()
```

Comment:
I copied this from https://github.com/matplotlib/matplotlib/pull/4290 so I assume it was done correctly by @tacaswell, but I find it odd that this accepts a `bbox` argument but doesn't do anything with it. Is that a mistake?

Reply:
makes sense. Should a docstring be added explaining something to that effect?

Reply:
I think I see how to implement more selective blitting for the ipympl frontend, but would/can the backend Agg renderer also needing blitting?

Reply:
The other consideration is that if any part of the image is transparent it will be forced to be a full image. That could also be optimized a bit with frontend blitting. Discussed far in the past here: https://github.com/matplotlib/matplotlib/issues/5419#issuecomment-154788744

Reply:
I'm sure it could be possible, but would require sending a different message probably. Doesn't have to be done in this PR though.

Reply:
Indeed I'm happy with this PR as is. Better blitting seems like a good 2021 goal.